morph/react/block-list.js

```diff
@@ -49,21 +50,45 @@ export function enter(node, parent, state) {
   }
 
   state.render.push(
-    `{Array.isArray(${value}) && ${value}.map((item, index, list) => `
+    `{Array.isArray(${value}) && ${value}.map((item, index, list) => {`
   )
 
   if (state.viewPath) {
     let itemDataContextName =
       getProp(node, 'itemDataContextName') ||
       defaultItemDataContextName(node, from)
+    let keys
+    if (hasProp(node, 'itemKeys')) {
```

Comment:
should we call this `itemKey` since at the end of the day it is identifying one key even though it can be a list of things?